opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/package/CMakeLists.txt

```diff
@@ -1,41 +1,102 @@
+    ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/setup.py
+    ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/LICENSE
+)
 
+add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/so.md5
+    DEPENDS ${PYTHON_LIB_NAME}
+    COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:${PYTHON_LIB_NAME}> ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/$<TARGET_FILE_NAME:${PYTHON_LIB_NAME}>
+    COMMAND md5sum ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/$<TARGET_FILE_NAME:${PYTHON_LIB_NAME}> > ${CMAKE_CURRENT_BINARY_DIR}/so.md5
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client
+    VERBATIM
+)
+add_custom_target(
+    python-opentera-webrtc-native-client-so
+    ALL
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/so.md5
+    VERBATIM
+)
 
-    install(FILES ${LIBRARY_OUTPUT_PATH}/opentera_webrtc_native_client.html DESTINATION python/package)
+# Generate html doc
+if(OPENTERA_WEBRTC_ENABLE_PYTHON_HTML_DOC)
+    add_custom_command(
+        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/${PYTHON_LIB_NAME}.html
+        DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/so.md5 ${GENERATE_CONFIGURE_OUTPUT_FILES} python-opentera-webrtc-native-client-so
+        COMMAND /usr/bin/env python3 -m pydoc -w ${PYTHON_LIB_NAME}
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client
+        VERBATIM
+    )
+    add_custom_target(
+        python-opentera-webrtc-native-client-html
+        ALL
+        DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/${PYTHON_LIB_NAME}.html
+        VERBATIM
+    )
+    set(HTML_DOC_DEPENDENCY_OPTION python-opentera-webrtc-native-client-html ${CMAKE_CURRENT_BINARY_DIR}/opentera_webrtc_native_client/${PYTHON_LIB_NAME}.html)
+else()
+    set(HTML_DOC_DEPENDENCY_OPTION ${CMAKE_CURRENT_BINARY_DIR}/so.md5 ${GENERATE_CONFIGURE_OUTPUT_FILES} python-opentera-webrtc-native-client-so)
 endif()
 
-# Install files
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.py DESTINATION python/package)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/README.md DESTINATION python/package)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/__init__.py DESTINATION python/package)
-install(FILES ../../../../LICENSE DESTINATION python/package)
-
-
-# Reinstall python library (could be better)
-install(TARGETS opentera_webrtc_native_client DESTINATION python/package)
-
-# Create python wheel package
-add_custom_target(python-wheel-package
-	COMMAND python3 setup.py bdist bdist_wheel sdist
-	WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/python/package)
-	
-# Signaling server files
-set(signaling_server_source_files
-    ../../../../signaling-server/signaling_server.py
-    ../../../../signaling-server/room_manager.py
-    ../../../../signaling-server/requirements.txt
-    ../../../../signaling-server/README.md)
-         
-install(FILES ${signaling_server_source_files} DESTINATION python/signaling-server)
```

Comment:
Removed for now, should probably be moved to its own CMakeLists.txt file if we want it to be installed with `make install`